backend/tracim_backend/lib/core/event.py

```diff
@@ -98,19 +99,43 @@ class EventBuilder:
 
     def __init__(self, config: CFG) -> None:
         self._config = config
-        self._current_user = None  # type: typing.Optional[User]
+        # NOTE S.G. 2020-06-04: need thread local storage as EventBuilder()
+        # is created only once/wsgi application
+        self._thread_local = threading.local()
```

Comment:
I'm not sure to understand why we do need thread local code, was it no more simple to ensure we hare not sharing object between thread ?  It maybe not possible but it's unclear to me yet.

Anyway, do this may solve issue #3064 ?

Reply:
There is no thread local part anymore: pending events are now stored on the given TracimContext instance.